src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.html

```diff
@@ -21,7 +21,7 @@
         </div>
         <div>
             <div class="btn-group btn-group-toggle" ngbRadioGroup [(ngModel)]="showAllEvents" name="radioBasic">
-                <label class="flat-button checkbox-push solid-grey" ngbButtonLabel *ngIf="timelineGenerator">
+                <label class="flat-button checkbox-push solid-grey" ngbButtonLabel *ngIf="!checkAllOption()">
```

Comment:
>  For this ngIf since pshowAllEvents is set with checkAllOptions() you can use that isntead.
To make it work I had to create another variable, since if we bind it to pshowAllEvents whenever its value changes the button would disappear.